test-cases/gutenberg/writing-flow/rich-text-formatting.md

```diff
@@ -11,76 +11,11 @@ Have a rich-text based component with content (Paragraph, Heading, Quote, Media
 
 - [Text color selection is not preserved when cursor is placed in the middle of text](https://github.com/WordPress/gutenberg/issues/42714)
 
-**Bold, Italic, strikethrough buttons**
-
-- Press the Bold button and write a bold word.
-- Press the Italic button and write an italic word.
-- Press strikethrough button and write strikethrough text.
-
-**Highlight selected text**
-
-- Write some text and select it.
-- Tap on the Highlight text button (A).
-- Select a color.
-- Expect to see the text with the selected color.
-
 **Highlight text without selection**
 
 - Without selecting any text tap on the highlight text button (A).
 - Select a color and write some text.
 - Expect to see the newly introduced text with the previously selected color.
-
```

Comment:
Nitpick: I'd leave an empty line between the "Highlight text without selection" test case and `TC007`.